### PR TITLE
Added license and usage instructions to readme

### PR DESCRIPTION
**Pull request by dgreenhalgh**
Finishes #8.

Depends on class names added in #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,129 @@
...
+```java
+int offsetPx = 10;
+int numColumns = 2;
+
+recyclerView.addItemDecoration(new GridBottomOffsetItemDecoration(offsetPx, numColumns));
+```
+
+###Project Setup
+
+**TODO**
```
Comment (dgreenhalgh) [inline README.md]: Issue here: #4

**Comment by dgreenhalgh** [comment]
Build fails because Travis isn't configured until #2.

**Comment by cstew** [comment]
Not sure how to test the code but the readme changes look good. :+1:

**Comment by matthew-compton** [comment]
:+1:

**Pull request merged**